### PR TITLE
Add a multi-GPU test for transformer backward.

### PR DESCRIPTION
**Pull request by wujingyue**
All tensors are replicated to all devices to start with. Future PRs will try to shard them. 

For #2199.

### CODE REVIEW

File: csrc/ops/alias.cpp
Comment (wujingyue) [inline csrc/ops/alias.cpp:1]: Ignore this file; extracted to #3178

File: csrc/ops/arith.cpp
Comment (wujingyue) [inline csrc/ops/arith.cpp:1]: ditto

File: nvfuser/__init__.py
Comment (wujingyue) [inline nvfuser/__init__.py:1]: Ignore this file; extracted to #3168.

**Comment by wujingyue** [comment]
!build

**Comment by wujingyue** [comment]
!build

**Comment by wujingyue** [comment]
!build

**Review by cowanmeg — APPROVED** [review]

**Comment by wujingyue** [comment]
!build

**Pull request merged**